pygtrie.py

```diff
@@ -1223,6 +1223,16 @@ def __init__(self, *args, **kwargs):
         self._separator = kwargs.pop('separator', '/')
         super(StringTrie, self).__init__(*args, **kwargs)
 
+    @property
+    def _separator(self):
+        return self.__separator
```

Comment:
@warenlg - see, I offered you this simple check yesterday, no need to go smart unless the maintainers ask you.